pupil-spa/src/app/services/check-start/check-start.service.spec.ts

```diff
@@ -47,12 +47,19 @@ describe('CheckStartService', () => {
   it('submit should call azure queue service successfully and audit successful call', async () => {
     const addEntrySpy = spyOn(auditService, 'addEntry');
     spyOn(tokenService, 'getToken').and.returnValue({ url: 'url', token: 'token'});
-    spyOn(azureQueueService, 'addMessage')
-      .and.returnValue(Promise.resolve());
+    // spyOn(azureQueueService, 'addMessage').and.returnValue(Promise.resolve());
+    let actualPayload;
+    spyOn(azureQueueService, 'addMessage').and
+    .callFake((queueName, url, token, payload, retryConfig) => {
+      actualPayload = payload;
+      return Promise.resolve();
```

Comment:
simple return should work too I believe.
Also you might want to remove the comment above.